todo_app/data/trello_items.py

```diff
@@ -121,44 +122,44 @@ def make_post_call_with_body_data(self):
         return requests.post(self.url, params=self.requestAuthPayload, data=self.body)
 
 
-class ViewModel:
-    def __init__(self, items):
-        self._items = items
+# class ViewModel:
```

Comment:
This is a lot of commented code! If we don't want to keep it, let's get rid of it rather than just commenting it out